README.md

```diff
@@ -1,20 +1,65 @@
-# Example documentation
+# Canonical documentation style guide
 
-This is an example of how to structure a set of [Markdown](https://daringfireball.net/projects/markdown/) documentation, for building with the [documentation builder](https://docs.ubuntu.com/documentation-builder/en/).
+This repository contains the documentation and the Vale rules for the documentation style guide.
 
-## Usage
+The style guide itself is written in Markdown and contained in the `en` directory
 
-To build documentation, you first need to install the documentation-builder:
+It is published online at: [docs.ubuntu.com/en/styleguide](https://docs.ubuntu.com/styleguide/en)
 
-``` bash
-snap install documentation-builder
-```
+## The Vale rules
+
+The Vale linter operates from a series of rules. These are defined in individual YAML files, grouped into 'Styles'.
+This repository contains the Canonical set of rules, or the Canonical Style. 
+
+### Adding to the rules
+
+Anyone is welcome to submit a PR to add additional rules. However, no additions will be considered unless they are part of the Canonical Style Guide as found at the website above.
+
+For information on how to create rules, see the Vale [documentation on Styles][Vale styles].
+
+### Using the rules
 
-Now simply run it inside this folder:
+The Vale rules are published here so that they can be used in any workflow anywhere. You can run Vale locally, as part of CI or in a GitHub workflow - all you need is Vale, a configuration file (which you can also copy from this repository) and the Canonical Styles. Two common scenarios are also catered for more directly here, as detailed below.
 
-``` bash
-documentation-builder
+## The Canonical Style GitHub action
+
+This repository also includes a file, `action.yml`, which is the basis of a GitHub action to automatically run Vale checks on incoming pull requests. 
+
+### Using the GitHub action in a workflow
+
+Your repository can make use of the action in a workflow. An example workflow is included in this repository and is demonstrated here. Note that the style guide action is merely part of a functioning workflow.
+
+```yaml
+on: [pull_request]
+
+jobs:
+  vale:
+    name: Style checker
+    runs-on: ubuntu-22.04
+    defaults:
+        run:
+            shell: bash
+            working-directory: .
+    steps:
+        - name: Checkout repo to runner
+          uses: actions/checkout@v3
+        - name: Install styles
+          uses: evilnick/test-action@v0
```

Comment:
Shouldn't this be the action from this repo?

Reply:
yes, it should be, but I can't change it usefully until the action is published here, and I can't publish it here until this lands - :chicken: <-> :egg:

Reply:
But this is not the action, it's the documentation - so it should reflect the state after the PR is merged.

Reply:
Okay, I made my best guess :shrug:. I suspect some of this is going to need to change anyhow since it will now be running from within a managed org.